GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.cs

```diff
@@ -185,32 +205,31 @@ private void Abort_Click(object sender, EventArgs e)
 
         private void contextMenuStrip1_Opening(object sender, CancelEventArgs e)
         {
-            if (GetSelectedRepo(sender, out var repo))
+            if (GetSelectedRepos(sender, out List<RecentRepoInfo?> repos))
             {
                 e.Cancel = false;
-                anchorToPinnedReposToolStripMenuItem.Enabled = repo.Repo.Anchor != Repository.RepositoryAnchor.Pinned;
-                anchorToAllRecentReposToolStripMenuItem.Enabled = repo.Repo.Anchor != Repository.RepositoryAnchor.AllRecent;
-                removeAnchorToolStripMenuItem.Enabled = repo.Repo.Anchor != Repository.RepositoryAnchor.None;
+                foreach (RecentRepoInfo repo in repos)
+                {
+                    anchorToPinnedReposToolStripMenuItem.Enabled = repo.Repo.Anchor != Repository.RepositoryAnchor.Pinned;
+                    anchorToAllRecentReposToolStripMenuItem.Enabled = repo.Repo.Anchor != Repository.RepositoryAnchor.AllRecent;
+                    removeAnchorToolStripMenuItem.Enabled = repo.Repo.Anchor != Repository.RepositoryAnchor.None;
+                }
             }
             else
             {
                 e.Cancel = true;
             }
         }
 
-        private bool GetSelectedRepo(object? sender, [NotNullWhen(returnValue: true)] out RecentRepoInfo? repo)
+        private bool GetSelectedRepos(object? sender, [NotNullWhen(returnValue: true)] out List<RecentRepoInfo?> repos)
```

Comment:
❔  It could output an enumerator. Though performance does not matter here.

Reply:
I just had a look and I don't think it's possible easily in fact...